telemetry/data_window.go

```diff
@@ -110,10 +109,7 @@ func readSQLMetric(timepoint time.Time, SQLResult *sqlUsageData) error {
 	promQL := "sum(tidb_executor_statement_total{}) by (instance,type)"
 	result, err := querySQLMetric(ctx, timepoint, promQL)
 	if err != nil {
-		if err1, ok := err.(*promv1.Error); ok {
-			return errors.Errorf("query metric error, msg: %v, detail: %v", err1.Msg, err1.Detail)
-		}
-		return errors.Errorf("query metric error: %v", err.Error())
+		logutil.BgLogger().Info("querySQLMetric got error")
```

Comment:
why not return `nil` here directly

Reply:
Thx for your suggestion, I will accept it!